content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: blog
+title: 'Gateway API First Beta'
+date: 2022-07-12
+slug: gateway-api-first-beta
+---
+
+**Author:** Shane Utt (Kong)
+
+We're pleased to announce that key Gateway API resources are graduating to
```

Comment:
Does “we“ mean Kong? If not, please clarify.

Reply:
```suggestion
Kubernetes SIG Networking is pleased to announce that key Gateway API resources are graduating to
```